### ISSUE TITLE
Proposal: Add github labels for scrum

### ISSUE THREAD

**Issue by tobbber** [amosproj/amos2021ws05-fin-prod-port-quick-check#5]
Proposal:
- Use custom github labels for Issue _status, type, priority_ and  _points_. 
- Add a milestone for each week with the submission deadline.
- Remove old labels
 
This would make it easier to use scrum in Github. See https://github.com/jvandemo/github-scrum-workflow for reference.

Label examples:
- `Status: blocked`  for issues that are blocked by another issue
- `Type: document`
